docs/src/content/docs/da/getting-started.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Kom godt i gang
+description: Lær, hvordan du kommer i gang med at bygge dit næste dokumentationssite med Starlight af Astro.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Starlight er et fuldt udstyret dokumentationstema bygget oven på [Astro](https://astro.build)-rammen. Denne vejledning vil hjælpe dig med at komme i gang med et nyt projekt. Se [manual installationsinstruktioner](/manual-setup/) for at tilføje Starlight til et eksisterende Astro-projekt.
```

Comment:
```suggestion
Starlight er et fuldt udstyret dokumentationstema bygget oven på [Astro](https://astro.build)-rammen. Denne vejledning vil hjælpe dig med at komme i gang med et nyt projekt. Se [manual installationsinstruktioner](/da/manual-setup/) for at tilføje Starlight til et eksisterende Astro-projekt.
```

- Is "manual installationsinstruktioner" correct? Only asking as it contains "manual installations" which is very close to the English version and want to make sure this is correct and not a typo.
- We want the link to point to the `/da/` page so that the user does not switch languages when following the link. If the page does not exist yet, Starlight fallback content will automatically show the English version with a warning saying it's fallback content. This also means that when this page get translated, the link should would be already correct.

Reply:
I think it's correct, but it may not be the best expression. If you have better words to replace it, please let me know.

Reply:
> If you have better words to replace it, please let me know.

As I mentioned in my review, I'm not a native speaker so I cannot really suggest a better word and was only asking to make sure it was correct and not a typo.

> I think it's correct, but it may not be the best expression.

If you are only thinking this is correct, may I ask if you are a native Danish speaker and if the translation may have been done using some sort of translation/AI tool? We are looking for translations at a [native speaker’s grammar level](https://contribute.docs.astro.build/guides/i18n/#quality-standards--adaptation) so if this is not the case, we would prefer to know.

Reply:
I am not a native Danish speaker. These translations were first translated using AI tools and then checked by Google Translate and DeepL. I think there will be no errors in accuracy. But if you require a native speaker to translate, I am sorry. Because I did not find this requirement on the translation page before.

In addition, if the page has been missing translation for a long time, I think it is not wrong to reserve an accurate translation first, but this is just my personal suggestion. If you must require extremely high translation standards, please close this PR directly.

Reply:
Thanks for your answer. We are indeed looking for native speakers to translate the documentation as technical terms can be quite tricky to translate as we have seen in the past or even in this PR. As Starlight provides [fallback content](https://starlight.astro.build/guides/i18n/#fallback-content) when a translation is missing, we prefer to have no translation at all than a potentially incorrect one and wait for a native speaker to provide a translation with more context/confidence.

We'll try to emphasize this requirement in our guidelines to avoid any potential confusion in the future.

Thank you for your understanding and trying to help with the translation nonetheless.